### DISCUSSION TITLE
[Bug]: Redirected to /xx when site is secured

### DISCUSSION THREAD

**Discussion by gizburdt** [beyondcode/herd-community#505] [General]
### Operating system version

macOs Sonoma 14.4

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.5.0

### PHP Version

8.3.3

### Bug description

First of all, awesome job on Herd!

All works fine. When I secure a site an want to visit it, it gets redirected to `https://example.org/dashboards/49` or `https://example.org/17`, or any other random number. So securing sites is not possible for me.

### Steps to reproduce

1. Install Herd
2. Park a folder
3. Secure a site
4. Visit the site

### Relevant log output

```shell
ERR_CONNECTION_REFUSED

Nginx and DNSMasq logs are empty
```

**Comment by mpociot** [comment]
We don't manually redirect after the site get secured, so this must be coming from somewhere in your application 🤔 
The only thing we do is redirect from `http://your-site` to `https://your-site`

**Comment by gizburdt** [comment]
Hmm, very strange! This redirect is done on all parked sites.

Maybe a conflict with something I installed? Not sure how to debug that.

**Comment by rynomars** [comment]
I am experiencing the same issue. Everything has been working just and and the am now getting a an error. TOO_MANY_REDIRECTS.

It is being redirect to /39. It is not ever hitting the code. I created an html file in the public directory and tried to access that file. I still get the redirect error on that file as well. @gizburdt did you every find a resolution to this issue?

**Comment by gizburdt** [comment]
No, I just work without SSL. The moment I activate SSL for a website, I get this problem.

**Comment by beneverard** [comment]
I'm getting the same issue as well, I have used Valet on its own before without issue, but now when I secure a site I'm redirected to a random /xx too.

And I too can confirm I don't have any redirects of this nature in my application.

@mpociot any thoughts on this? Thanks, Ben

**Comment by beneverard** [comment]
@gizburdt @rynomars I've had the same issue and investigated and have found the problem and how to solve it whilst the core bug is being investigated, I've added more details in issue #998
